### PR TITLE
Add small fixes to website

### LINKED ISSUE #165: 🐾 Smol Update: white link for activities pls
image
 
white link plsss says karen :))

* [ ] :wave: I may be able to implement this feature request

---

This is a :feet: smol update

### LINKED ISSUE #170: 🐾 Smol Update: page is way too fast to read on autoplay
alot of the people slides are way too fast to read on autoplay

image

maybe slow animation? idk

* [ ] :wave: I may be able to implement this feature request

---

This is a :feet: smol update

### LINKED ISSUE #173: 🐾 Smol Update: DOC update
image

update DOC stuff
- this link links.uclaacm.com/dayofcodepassport
- new word: C O D I N G

* [ ] :wave: I may be able to implement this feature request

---

This is a :feet: smol update

### LINKED ISSUE #179: 🧹 Cleanup: renaming Activity3 game stuff
it's a little hard to follow the game slide names sometimes

### Proposed Solution
- rename so all names are consitent with their states here https://github.com/uclaacm/Playnet/blob/217f922ba86f31a941bd467c3d1bc0048387fc97/src/components/Activities/Activity3/Game/index.tsx#L178-L197
- [x] rename `FeaturesTutorial` to `PriorityExplanation` 
- [x] rename `FeatureSlidebar` to `PriorityWeighing` 
- [x] rename `ABTestDescription` to `ABTestingExplanation` (or vice versa) or just make them the same, anything is good
- [x] rename `ABFinalReport` to `FinalReport`

tbh anything is good as long as they are the same or somethign

* [ ] :wave: I may be able to implement this cleanup request

---

This is a :broom: cleanup request

### PR DESCRIPTION
**Pull request by darren2hang**
- Change link on bottom of activities page to white instead of blue
- Increase animation time on People lecture animation to allow user time to read text
- Update Day of Code page's link and code word
- Rename Components in Activity 3 Game to match the names in the GameConstants file

closes #165
closes #170 
closes #173 
closes #179

### CODE REVIEW

File: src/components/Activities/Activity3/index.tsx [hunk 1]
```diff
@@ -91,12 +91,12 @@ function Activity3(): JSX.Element {
     {
```
Comment (reginawang99) [inline src/components/Activities/Activity3/index.tsx:91]: can you change line 89 to like 3.5 or 4?

File: src/components/DayOfCode.tsx [hunk 2]
```diff
@@ -5,19 +5,19 @@ import { HeaderSections } from './shared/PlaynetConstants';
...
   return (
     <Base section={HeaderSections.DAY_OF_CODE}>
       <div id={'feedback-container'}>
         <p>
           The code for this event is:
-          <h1><i>C O D</i></h1>
+          <h1><i>C O D I N G</i></h1>
           <button className='playnet-button'>
             <a href={link_to_checklist} target='_blank' rel='noreferrer'>Add it to your passport!</a>
           </button>
```
Comment (reginawang99) [inline src/components/DayOfCode.tsx:17]: out of scope of this pr, but could you help add this in? thanks!!
```suggestion
            <a href={link_to_checklist} className='playnet-button' id='feedback-link'  target='_blank' rel='noreferrer'>Add it to your passport!</a>
```

**Review by reginawang99 — CHANGES_REQUESTED** [review]
i am a nitpick ty for fix

File: src/components/Activities/Activity3/index.tsx (hunk 1 above)
Comment (darren2hang) [reply src/components/Activities/Activity3/index.tsx:91]: Sure!!
